doc/governance.rst

```diff
@@ -0,0 +1,95 @@
+
+Project Leader
+--------------
+
+The Ceph project is currently led by Sage Weil <sage@redhat.com>.  The
+project leader is responsible for guiding the overall direction of the
+project and ensuring that the developer and user communities are
+healthy.
+
+
+Committers
+----------
+
+Committers are project contributors who have write access to the central
+Ceph code repositories, currently hosted on GitHub.  This group of developers
+is collectively empowered to make changes to the Ceph source code.
+
+Generally speaking, no individual should make a change in isolation:
+all code contributions go through a collaborative review process (and
+undergo testing) before being merged.  The specifics of this process
+are dynamic and evolving over time.
+
+New committers are added to the project (or committers removed from
+the project) at the discretion of the Ceph Leadership Team (below).
+The criteria for becoming a contributor include a consistent level of
+quality and engagement in the project over time.
+
+
+Ceph Leadership Team
+--------------------
+
+The Ceph Leadership Team (CLT) is a collection of component leads and
+other core developers who collectively make technical decisions for
+the project.  These decisions are generally made by consensus,
+although voting may be used if necessary.
+
+The CLT meets weekly via video chat to discuss any pending issues or
+decisions.  Minutes for the CLT meetings are published at
+`http://pad.ceph.com/p/clt-weekly-minutes`_.
+
+Committers are added to or removed from the CLT at the discretion of
+the CLT itself.
+
+Current CLT members are:
+
+ * Ahbishek Lekshmanan <abhishek@suse.com>
+ * Alfredo Deza <adeza@redhat.com>
+ * Gregory Farnum <gfarnum@redhat.com>
+ * Jason Dillaman <dillaman@redhat.com>
+ * Josh Durgin <jdurgin@redhat.com>
+ * Joao Eduardo Luis <joao@suse.de>
+ * John Spray <john.spray@redhat.com>
+ * Matt Benjamin <mbenjami@redhat.com>
+ * Myoungwon Oh <omwmw@sk.com>
+ * Ken Dreyer <kdreyer@redhat.com>
+ * Haomai Wang <haomai@xsky.com>
+ * Lenz Grimmer <lgrimmer@suse.com>
+ * Sage Weil <sage@redhat.com>
+ * Xie Xingguo <xie.xingguo@zte.com.cn>
+ * Yehuda Sadeh <yehuda@redhat.com>
+ * Zack Cerza <zcerza@redhat.com>
+
+Component Leads
+---------------
+
+Each major subcomponent of the Ceph project has a lead engineer who is
+responsible for guiding and coordinating development.  The leads are
+nominated or appointed at the discretion of the project leader or the
+CLT.  Leads responsibilities include:
+
+ * guiding the (usually) daily "stand-up" coordination calls over video chat
+ * building the development roadmap for each release cycle
+ * coordinating development activity between contributors
+ * ensuring that contributions are reviewed
+ * ensuring that different proposed changes do not conflict
+ * ensuring that testing remains robust (new features include tests, changes do not break tests, etc.)
+
+All component leads are included on the CLT.
```

Comment:
How about adding a small comment about what the component lead's role within the CLT looks like, e.g. reporting progress/status updates related to the sub-component or acting on behalf of the developers working on the respective component?